squad/core/notification.py

```diff
@@ -61,7 +61,13 @@ def summary(self):
 
     @property
     def recipients(self):
-        return [r.email for r in self.project.subscriptions.all()]
+        emails = []
+        for sub in self.project.subscriptions.all():
+            if sub.user and sub.user.email:
+                emails.append(sub.user.email)
+            else:
+                emails.append(sub.email)
+        return emails
```

Comment:
This duplicates the logic in `Subscription.__str__`. I think this logic should be in a method of `Subscription` (e.g. `get_email`), and both this code and `Subscription.__str__` should use it.